contributors/devel/cri-validation.md

```diff
@@ -0,0 +1,50 @@
+# CRI Validation Testing
+
+CRI validation testing provides a series of common validation test suites for Kubelet CRI (_Container Runtime Interface_).
+
+It helps container runtime maintainers to validate their runtime when integrating with kubelet CRI.
+
+*Note: No kubelet is running. The test suites run directly targeting the shim of container runtime.*
+
+*Note: CRI validation testing is alpha and maintained at [cri-tools](https://github.com/kubernetes-incubator/cri-tools) repo.*
```

Comment:
Remove `*Note`

Suggestion:
"CRI validation testing is currently Alpha and is hosted at the [cri-tools](https://github.com/kubernetes-incubator/cri-tools) repository. We encourage the CRI developers to report bugs or help extend the test coverage by adding more tests."

Please modify it according to your knowledge on the current test coverage and the limitation of the test suite.